posts/2020-09-29-microprofile-openapi-beta-200011.adoc

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+title: "New features and loads of improvements for Jakarta EE 9 plus MicroProfile OpenAPI 2.0 in Open Liberty 20.0.0.11 beta"
```

Comment:
Please do not leave the dash out of the version ... `20.0.0.11-beta` NOT `20.0.0.11 beta`. That applies throughout, in any and all docs.

Reply:
Done